examples/website/map-tile/app.js

```diff
@@ -68,14 +83,26 @@ export default function App({showBorder = false, onTilesLoad = null}) {
     }
   });
 
+  const renderCopyrightAndLicense = () => (
+    <div style={COPYRIGHT_LICENCE_STYLE}>
+      {'© '}
+      <a style={LINK_STYLE} href="http://www.openstreetmap.org/copyright" target="blank">
+        OpenStreetMap
+      </a>
+      {' contributors'}
```

Comment:
I think the whole text should be a link, not just the word `OpenStreetMap`

Reply:
On OSM's [own map](https://www.openstreetmap.org/#map=5/38.007/-95.844) the hyperlink includes `contributors` but not the copyright symbol 🤷‍♂️ 
![image](https://user-images.githubusercontent.com/15164633/126525099-af40cb6f-cdc7-441e-93f4-cab9bc25e618.png)

Reply:
Done